src/library/include/platform/timer.h

```diff
@@ -59,4 +59,5 @@ static inline void platform_dai_timestamp(struct comp_dev *dai,
 static inline void platform_dai_wallclock(struct comp_dev *dai,
 	uint64_t *wallclock) {}
 
+extern struct timesource_data platform_generic_queue[];
```

Comment:
you have probably mistaken the timer.h file, thats why all builds failed,
try this one in platform folders and it will work
or another one if fits better

Reply:
@abonislawski Thank you very much for this. It's my fault and the PR is updated.